package.json

```diff
@@ -34,6 +34,8 @@
     "react": "^0.13.0"
   },
   "dependencies": {
+    "create-react-class": "^15.5.1",
+    "prop-types": "^15.5.6",
```

Comment:
I'm not sure. I guess you would pull in `react` as a peer dependency as users would actively want to provide their own copy of it, but is that the case with these two? `react` they definitely would have installed anyway, but might not use `prop-types` or `create-react-class`, which they then have to install manually and include in their project dependencies (as it goes with `peerDependencies`). Can you explain why you think they should be `peerDependencies`?